worlds/sm/Options.py

```diff
@@ -62,39 +65,46 @@ class MaxDifficulty(Choice):
     default = 4
 
 class MorphPlacement(Choice):
+    """Influences where the Morphing Ball with be placed."""
     display_name = "Morph Placement"
     option_early = 0
     option_normal = 1
     default = 0
 
 class StrictMinors(Toggle):
+    """Instead of using the Minors proportions as probabilities, enforce a strict distribution to match the proportions as closely as possible."""
     display_name = "Strict Minors"
 
 class MissileQty(Range):
+    """The higher the number the higher the probability of choosing missles when placing a minor."""
     display_name = "Missile Quantity"
     range_start = 10
     range_end = 90
     default = 30
 
 class SuperQty(Range):
+    """The higher the number the higher the probability of choosing super missles when placing a minor."""
     display_name = "Super Quantity"
     range_start = 10
     range_end = 90
     default = 20
 
 class PowerBombQty(Range):
+    """The higher the number the higher the probability of choosing power bombs when placing a minor."""
     display_name = "Power Bomb Quantity"
     range_start = 10
     range_end = 90
     default = 10
 
 class MinorQty(Range):
+    """From 7%: minimum number of minors required to finish the game to 100%: all minors locations contain a minor (vanilla like)."""
```

Comment:
Maybe remove "all minors locations contain a minor (vanilla like)". That's not the case in AP. "Majors Split" is always set to full.